pkg/webui/console/components/mac-settings-section/index.js

```diff
@@ -469,43 +461,71 @@ const MacSettingsSection = props => {
         />
       </Form.FieldContainer>
       <Form.Field
-        name="mac_settings.use_adr"
-        component={Checkbox}
-        label={m.useAdr}
-        onChange={handleUseAdrChange}
+        name="mac_settings.adr"
+        component={Radio.Group}
+        title={'Adaptive data rate (ADR)'}
```

Comment:
Shouldn't this be internationalized as well ?

Reply:
True, I overlooked these. Will fix